Prepare bam 20.04.9 (#1256)
centreon/centreon-documentation: en/releases/centreon-commercial-extensions.md

## Centreon BAM

### 20.04.9

> `January 07, 2022`

Reviewer: ```suggestion
Release date: `January 14, 2022`
```

#### Bugfixes

> - [Configuration] Fixed an error when generating config if a disabled BA is connected to an escalation task

Reviewer 2: ```suggestion
- [Configuration] Fixed an error when generating poller configuration with a disabled BA linked to an escalation task
```

> - [Configuration] Fixed the relations between BA and escalations are exported in RS engine conf regardless the BA is linked to the RS or not

Reviewer 2: ```suggestion
- [Configuration] BA escalation configuration is not exported to Remote Server anymore if the BA is not linked to this Remote Server
```

> - [Monitoring] Fixed Business Activities widget height doesn't adapt

Reviewer 2: ```suggestion
- [Monitoring] Business Activities widget now scales to the number of BAs
```
